packages/services/f-wdio-utils/src/page.object.js

```diff
@@ -1,7 +1,7 @@
-const { source } = require('axe-core');
-const { buildUrl } = require('./storybook-extensions');
+import { source } from 'axe-core';
+import { buildUrl } from './storybook-extensions';
 
-class Page {
+export default class Page {
```

Comment:
As per my slack comment:
```suggestion
export default class Page {
    constructor (componentType, componentName, componentSelector) {
        this.title = 'Component URLS';
        this.componentType = componentType;
        this.componentName = componentName;
        this.componentSelector = componentSelector;
        this.path = '';
    }

    get component () {
        return $(`[${this.componentSelector]`);
    }
```

Reply:
Will refactor f-wdio-utils in a later PR as there's some other changes we need to make, but can't until tests are refactored to async/await.